### DISCUSSION TITLE
Kontakt do regionlanych liderów meetupów, propozycje sesji ?

### DISCUSSION THREAD

**Discussion by JanuszNowak** [AzureCommunityPL/group#386] [Ogólna]
Hej, mam spis kontaktów z kim można się kontaktować w celu zgłaszani sesji, organizacji meetupów w poszczególnych regionach ?

**Comment by mgrabarz** [comment]
Pytasz, czy mamy taki spis? 
Chyba oficjalnie nie, ale też niewiele meetupów się obecnie odbywa - chyba tylko białostocki organizowany przez Rafała Hryniewskiego i Marcina Iwanowskiego.

**Reply by mmisztal1980** [reply]
w Warszawie staramy się o organizować WG.NET w formie hybrydowej, więcj jakby co to zapraszam - nie samym .NET'em człowiek żyje
